test/src/main/java/org/apache/accumulo/test/manager/SuspendedTabletsIT.java

```diff
@@ -322,6 +325,20 @@ private void suspensionTestBody(TServerKiller serverStopper) throws Exception {
         Thread.sleep(1000);
         ds = TabletLocations.retrieve(ctx, tableName);
       }
+
```

Comment:
I currently am in the process of trying to understand this test.  While doing so I added the following at this loacation and ran it locally.

```java
log.info("hc:{} sc:{} ac:{}", ds.hostedCount, ds.suspendedCount, ds.assignedCount);
```

I am seeing this output the following.  So even before the tablet server is stopped the suspend count is already zero.

```
hc:30 sc:0 ac:0
```

Reply:
Interesting, I'll look into this

Reply:
I was able to reproduce your log information. Looking now at where `deleteSuspension` is called, it's being called in `MetadataStateStore.setFutureLocations` and `MetadataStateStore.setLocations`. The changes to this test likely don't need to be merged. Also, I think #3314 can be closed without changes as the code in the description is not reflective of the current state of the code now.